controllers/users.js

```diff
@@ -1,13 +1,40 @@
 
 const userQuery = require('../models/users')
 const imageService = require('../services/imageService')
+const fetch = require('node-fetch')
 /**
  * Fetches the data about our users
  *
  * @param req {Object} - Express request object
  * @param res {Object} - Express response object
  */
 
+const verifyUser = async (req, res) => {
+  try {
+    const userId = req.userData.id
+    if (!req.userData.identityURl) {
```

Comment:
```suggestion
    if (!req.userData?.identityURl) {
```

Reply:
We have to append a Question Mark(?) here ?

Reply:
The Build Process is also failing due to some reason